.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,40 @@
+---
+name: 🐛 Bug Report
+about: Submit a bug report to help us improve
+labels: "bug"
+---
+
+## 🐛 Bug Report
+
+(A clear and concise description of what the bug is)
+
+## To Reproduce
+
+(Write your steps here:)
+
+## Expected behavior
+
+<!--
+  How did you expect your project to behave?
+  It’s fine if you’re not sure your understanding is correct.
+  Write down what you thought would happen.
+-->
+
+(Write what you thought would happen.)
+
+## Actual Behavior
+
+<!--
+  Did something go wrong?
+  Is something broken, or not behaving as you expected?
+  Describe this section in detail, and attach screenshots if possible.
+  Don't only say "it doesn't work"!
+-->
+
+(Write what happened. Add screenshots, if applicable.)
+
+## Your Environment
+
+<!-- Include as many relevant details about the environment you experienced the bug in -->
+
+(Write Environment, Operating system and version etc.)
```

Comment:
You could add this check and probably add it to all the issue templates

### Have you spent some time to check if this issue has been raised before?
<!-- Have you googled for a similar issue or checked our older issues for a similar bug -->